src/base/utils/string.cpp

```diff
@@ -177,8 +177,12 @@ QString Utils::String::fromStdString(const std::string &str)
 
 std::string Utils::String::toStdString(const QString &str)
 {
+#ifdef QBT_USES_QT5
```

Comment:
When we has dropped Qt4 support we could use QString::toStdString directly... Isn't it?